packages/applicant/src/pages/submissions/[submissionId]/summary.page.tsx

```diff
@@ -34,16 +37,35 @@ export const getServerSideProps: GetServerSideProps<
   const jwt = getJwtFromCookies(req);
   const submissionId = params.submissionId.toString();
 
-  const { sections, grantSubmissionId, applicationName, grantSchemeId } =
-    await getSubmissionById(submissionId, getJwtFromCookies(req));
+  const {
+    sections,
+    grantSubmissionId,
+    applicationName,
+    grantSchemeId,
+    submissionStatus,
+  } = await getSubmissionById(submissionId, getJwtFromCookies(req));
+
+  const submissionExists =
+    submissionStatus === 'IN_PROGRESS' || 'SUBMITTED' || 'GRANT_CLOSED';
+
+  if (!submissionExists) {
+    return {
+      redirect: {
+        destination: `/grant-is-closed`,
+        permanent: false,
+      },
+    };
+  }
```

Comment:
I don't think we need this check - if the submission does not exist `getSubmissionById` will throw an error which will render a whitescreen rather than render any incorrect content. `closedAndInProgress` below can just be a check for if the application is removed